apps/condo/domains/ticket/utils/clientSchema/Renders.tsx

```diff
@@ -107,7 +107,7 @@ export const getClassifierRender = (intl, search: FilterValue) => {
         const placeClassifier = get(record, ['placeClassifier', 'name'])
         const postfix = `\n(${placeClassifier})`
 
-        return getTableCellRenderer(search, true, postfix, null, POSTFIX_PROPS)(text)
+        return getTableCellRenderer(search, false, postfix, null, POSTFIX_PROPS)(text)
```

Comment:
please, add screenshots 
1. before ...
2. after ...

Reply:
1. Before : 
![image](https://user-images.githubusercontent.com/64303474/167997843-989b17cc-a381-4d95-981c-34235eceb3ed.png)
2. After: 
![image](https://user-images.githubusercontent.com/64303474/167997923-87f5a29c-f313-43e2-b312-3a293c94c2ed.png)

Reply:
In a issue description :)